src/components/image-with-text-block/image-with-text-block.vue

```diff
@@ -87,6 +98,10 @@
     background-color: var(--white);
   }
 
+  .image-with-text__body-text .structured-text__blue-text {
+    text-align: left;
```

Comment:
Not ideal I know, here are some other options:

- Remove the `Structured Text - Blue Text` from the blocks and make all text next to the image blue, that can be done without referring to the `.structured-text...`.
But technically we have one case were the text next to the image isn't blue /work-at 
![image](https://github.com/voorhoede/voorhoede-website/assets/15196342/308fd482-c1fd-4da6-91f2-61681cbcb5b6)

- Afterwards, on another PR, add a `Text alignment` field to the `Structured Text - Blue Text` instead of assuming it to be always centered

Reply:
Sorry I overlooked the last bit of your comment. Let's proceed with that in a separate PR indeed! I've created one: https://trello.com/c/y64w5KYH/694-add-text-alignment-option-to-structured-text-blue-text